### DISCUSSION TITLE
MSECS%40JHU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#195] [Announcements]
# MSECS%40JHU/

None

https://opencs.app/MSECS%40JHU/

**Comment by TadejPogacar1** [comment]
这个项目定位只有B吗？

**Reply by xichenpan** [reply]
根据目前有限DataPoints数据，难度不会太高，个人倾向在B到B+之间

**Comment by XXXJumpingFrogXXX** [comment]
感觉难度应该在B+到A-之间

**Reply by charlesyangxinye** [reply]
JHU MSECS bar不高的主要原因是不是巴尔的摩比较危险啊。

**Reply by XXXJumpingFrogXXX** [reply]
但感觉难度也不小的 自己问了下往届的学长学姐 感觉比CS28难度要大

**Comment by FeijiangHan** [comment]


**Reply by vince76213** [reply]
可是我看地里的dp也大多数是3.8 3.9的... 可能地里愿意汇报背景的bg都还不错？

**Comment by vince76213** [comment]
我整理一下地里的dp (24Fall) 我觉得bg都是不错的，希望大家补充一些低背景dp，避免bias.
1）国内C2本 CS gpa88.7 3.8/4.3 1华为2小公司实习 1二区靠后作者 1普刊二作 G330
2）武汉985 gpa3.8+ 1暑研 1实习
3）港本 gpa3.91 5%
4）中9 gpa3.9 1二区一作 1实习
5）海本 gpa3.89 G327 Meta Teammatch实习
6）华五本 gpa3.87 美国ap强推 无pub

**Reply by vince76213** [reply]
7）顶尖211 软工 gpa 3.7 3无产出科研 G327
8）末9 gpa3.96 1大厂后端 3水科研 1交叉方向n作pub

**Comment by Leadlegend** [comment]
bar连续两年上涨基本是肯定的...身边有top2cs rank50%的最后准备去，不过确实比较random，对非科班或低bg也不会硬卡死。

**Comment by mrvgao** [comment]
Hopkins在读，学生背景水平差异非常大。总体来说，underrepresented 群体录取Bar明显低很多。例如被录取的学生中，女生的水平明显远低于男生。并且最近几年新招收的教授也有该趋势，最近新加入的一些女老师，如果背景为男，大概率申请不到Hopkins的Phd或者Postdoc。 但这件事情也是一些好事，个人认为，毕竟以我个人生活经历而言，从小女孩子就会常常被告知理工科学不好，逻辑思维不如男生云云。Hopkins这样也算是“矫枉”吧，我理解。
